active/0001-automated-releases.md

```diff
@@ -0,0 +1,97 @@
+- Automated Releases
+- @kleimkuhler, @alpeb
+- 2020-02-17
+- 2020-03-03
+- RFC PR: [linkerd/rfc#7](https://github.com/linkerd/rfc/pull/7)
+- Linkerd Issue: [linkerd/linkerd2#3021](https://github.com/linkerd/linkerd2/issues/3021)
+- Reviewers: @grampelberg
+
+# Summary
+
+[summary]: #summary
+
+The goal for this project is to make the Linkerd2 release process as automated
+as possible. The process currently requires that the responsible member take
+multiple manual steps in order for the release artifacts to be properly built,
+tested, and updated for installations.
+
+Instead of addressing these points individually, the release process should be
+re-evaluated as a whole in order to utilize the CI workflow to automate the
+process where possible.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+The manual steps required in order to perform a release can lead to any number
+of errors by fault of the responsible member. These include errors such as: 
+- forgetting to perform a particular step
+- completing the steps in an incorrect order
+- dealing with flakiness related to the timing of when steps are completed
+
+Additionally, the member must "babysit" the individual steps in order to start
+the next one. This can be time consuming.
+
+Once implemented, the release process will be:
+1. The responsible member opens a pull request that updates `CHANGES.md` with
+   the relevant changes for the upcoming release.
+2. Once step `1` is merged, the member pushes a signed tag for the release.
+3. A workflow is triggered that builds the docker images and tags them with
+   the step `2` release tag. Integration tests are run. If tests pass, a
+   GitHub release and created the the release artifcats are uploaded. The
+   hard-coded Linkerd version in [website](https://github.com/linkerd/website)
+   is automatically updated so that the install script points to the release
+   artifacts.
+4. The member sends out CNCF announcement emails.
+
+# Design proposal (Step 2)
```

Comment:
This will go away in the final doc, right?

Reply:
No now that #8 has merged, `Design Proposal` is a separate step. Maybe this could just be added to `design/` right away, but I'd like to keep this on a "normal" RFC path which first should start as being added to `problem/`.